README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
We could do that ... but we prefer to link to the symfony.com logo because it gives us more control and flexibility (we could change it for some special occasion, etc.)